### PR TITLE
Landing page

### LINKED ISSUE #17: Landing Page and API Fetch Integration
Description: 
Update index.html to include a form where the user can input a project ID and submit it. On submit, fetch the project data from the API and redirect to the RBAC page if the project exists.

Acceptance Criteria:

- [ ] Form in index.html that accepts a project ID.

- [ ] Submits the project ID and fetches project data from https://api.t-pen.org/project/${projectId}.

- [ ] If successful, redirects to rabc.html.

- [ ] If failed, shows an error message.

### PR DESCRIPTION
**Pull request by Matthew-Clendenning**
Fixes #17 

What was changed: Fixed an error where the landing page wasn't styled correctly. Created a file to handle user input where the user inputs a project ID and displays the project details on another page.

Why it was changed: These changes are needed because specific users need to be able to view the details of a project they want to manipulate.

How it was changed: Created a JavaScript file where it fetches the project data from the API if the project exists. If it does exist it displays the data on another html page, otherwise it will give an error on the page. The functionality isn't quite sound and receives a 500 internal server error.

### CODE REVIEW

**Review by MutyaSowmya123 — CHANGES_REQUESTED** [review]
- remove menutoggle.js we don't need that at present.

- Also revert the changes done in test.html file.

Everything else seems good, work on these changes for now, I waiting for some feedback from our client. So based on that I may add some more changes further.

File: components/classroom/fetchData.mjs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
...cGVuIl0sImh0dHA6Ly9kdW5iYXIucmVydW0uaW8vYXBwX2ZsYWciOlsidHBlbiJdLCJodHRwOi8vcmVydW0uaW8vdXNlcl9yb2xlcyI6eyJyb2xlcyI6WyJkdW5iYXJfdXNlcl9wdWJsaWMiLCJnbG9zc2luZ191c2VyX3B1YmxpYyIsImxyZGFfdXNlcl9wdWJsaWMiLCJyZXJ1bV91c2VyX3B1YmxpYyIsInRwZW5fdXNlcl9wdWJsaWMiXX0sImh0dHA6Ly9kdW5iYXIucmVydW0uaW8vdXNlcl9yb2xlcyI6eyJyb2xlcyI6WyJkdW5iYXJfdXNlcl9wdWJsaWMiLCJnbG9zc2luZ191c2VyX3B1YmxpYyIsImxyZGFfdXNlcl9wdWJsaWMiLCJyZXJ1bV91c2VyX3B1YmxpYyIsInRwZW5fdXNlcl9wdWJsaWMiXX0sImlzcyI6Imh0dHBzOi8vY3ViYXAuYXV0aDAuY29tLyIsInN1YiI6ImF1dGgwfDY3MDk4NjczOTJlOTg1NDY5Yzc5OWY1YyIsImF1ZCI6WyJodHRwczovL2N1YmFwLmF1dGgwLmNvbS9hcGkvdjIvIiwiaHR0cHM6Ly9jdWJhcC5hdXRoMC5jb20vdXNlcmluZm8iXSwiaWF0IjoxNzI4ODUxNTcyLCJleHAiOjE3Mjg4NTg3NzIsInNjb3BlIjoib3BlbmlkIHByb2ZpbGUgZW1haWwgdXBkYXRlOmN1cnJlbnRfdXNlcl9tZXRhZGF0YSBvZmZsaW5lX2FjY2VzcyIsImF6cCI6ImJCdWdGTVdIVW8xT2huU1pNcFlVWHhpM1kxVUpJN0tsIn0.4jcJCJkc9bxo8VLaRhSbuB9fQxlOurhXE82tZ26TDU5Vp7QR7Y8qxBSm8f7hjvJe48gf4gvZD68KXCUabezAqlVe5MuUTLSnhmqfEHV5VJTmeIBmljCyrUowvrZWlzEMWV3EjdC-Cq4t8jVeFtr9DTOsHqx9k1sRaKIrkCCN1Ww8LFWqgW4ycL7cLg6lVARDKSjec-NB8kQwXPo-eaOPAcEgizUxx91teVufmrFX_9h8eR5HvdhEd9BWKdqGC9Ezjfqa1NLaJ_zDzsNe73xLgTH6BaO2EMPfvXw8ZWOhQdBGNQ_vGhCaU2WiOgeK7H24q9V2nhpGKWjCiWmGtdM8PA';
```
Comment (kate-holdener) [inline components/classroom/fetchData.mjs:4]: Generally, it's not a good idea to have tokens in a GitHub repository. @Matthew-Clendenning @MutyaSowmya123 was this an intentional change?

File: components/classroom/groups/api/fetchProjectData.mjs [hunk 2]
```diff
@@ -34,4 +34,38 @@ export async function fetchProjectData(projectId) {
...
...cGVuIl0sImh0dHA6Ly9kdW5iYXIucmVydW0uaW8vYXBwX2ZsYWciOlsidHBlbiJdLCJodHRwOi8vcmVydW0uaW8vdXNlcl9yb2xlcyI6eyJyb2xlcyI6WyJkdW5iYXJfdXNlcl9wdWJsaWMiLCJnbG9zc2luZ191c2VyX3B1YmxpYyIsImxyZGFfdXNlcl9wdWJsaWMiLCJyZXJ1bV91c2VyX3B1YmxpYyIsInRwZW5fdXNlcl9wdWJsaWMiXX0sImh0dHA6Ly9kdW5iYXIucmVydW0uaW8vdXNlcl9yb2xlcyI6eyJyb2xlcyI6WyJkdW5iYXJfdXNlcl9wdWJsaWMiLCJnbG9zc2luZ191c2VyX3B1YmxpYyIsImxyZGFfdXNlcl9wdWJsaWMiLCJyZXJ1bV91c2VyX3B1YmxpYyIsInRwZW5fdXNlcl9wdWJsaWMiXX0sImlzcyI6Imh0dHBzOi8vY3ViYXAuYXV0aDAuY29tLyIsInN1YiI6ImF1dGgwfDY2YjliMzE4YzZhMjU4MDZjOTgxMDg3YyIsImF1ZCI6WyJodHRwczovL2N1YmFwLmF1dGgwLmNvbS9hcGkvdjIvIiwiaHR0cHM6Ly9jdWJhcC5hdXRoMC5jb20vdXNlcmluZm8iXSwiaWF0IjoxNzI4NjU5NTgyLCJleHAiOjE3Mjg2NjY3ODIsInNjb3BlIjoib3BlbmlkIHByb2ZpbGUgZW1haWwgdXBkYXRlOmN1cnJlbnRfdXNlcl9tZXRhZGF0YSBvZmZsaW5lX2FjY2VzcyIsImF6cCI6ImJCdWdGTVdIVW8xT2huU1pNcFlVWHhpM1kxVUpJN0tsIn0.QAdI3o_MQLZ_pqs6-pSplqKXdq6xXYaMS1QRG6xrVKAieFFHZL67V9uTRgRSMPOz4UxG83KzXXMqBbVV83jc79zlbBidUfej-1HVOYMIYGikImazl9Mt-JFCjLffSovM2PIv68tieqcFjXplJaTpdWQviMXqTlC9xeXGgpnKJ_XpdqZX5A19bLzy3yF2AvqYC9lPD79tuE5bV1t0rYL5B3doQ0NdajNcaklyZgiBqP2lFG4mDXKFtWHC_hA4z6DTYx2QaH5fvIovdrA8y3EwTk7Z4-8lEpqqgut4aLwj0Q4nkdMjR4rtrtvRVjlPhLU9_BYrX22J2R1p4nDC7QvCcw";
```
Comment (kate-holdener) [inline components/classroom/groups/api/fetchProjectData.mjs:40]: Generally, it's not a good idea to have tokens in a GitHub repository. @Matthew-Clendenning @MutyaSowmya123 was this an intentional change?

File: components/classroom/main.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
...0cGVuIl0sImh0dHA6Ly9kdW5iYXIucmVydW0uaW8vYXBwX2ZsYWciOlsidHBlbiJdLCJodHRwOi8vcmVydW0uaW8vdXNlcl9yb2xlcyI6eyJyb2xlcyI6WyJkdW5iYXJfdXNlcl9wdWJsaWMiLCJnbG9zc2luZ191c2VyX3B1YmxpYyIsImxyZGFfdXNlcl9wdWJsaWMiLCJyZXJ1bV91c2VyX3B1YmxpYyIsInRwZW5fdXNlcl9wdWJsaWMiXX0sImh0dHA6Ly9kdW5iYXIucmVydW0uaW8vdXNlcl9yb2xlcyI6eyJyb2xlcyI6WyJkdW5iYXJfdXNlcl9wdWJsaWMiLCJnbG9zc2luZ191c2VyX3B1YmxpYyIsImxyZGFfdXNlcl9wdWJsaWMiLCJyZXJ1bV91c2VyX3B1YmxpYyIsInRwZW5fdXNlcl9wdWJsaWMiXX0sImlzcyI6Imh0dHBzOi8vY3ViYXAuYXV0aDAuY29tLyIsInN1YiI6ImF1dGgwfDY2YjliMzE4YzZhMjU4MDZjOTgxMDg3YyIsImF1ZCI6WyJodHRwczovL2N1YmFwLmF1dGgwLmNvbS9hcGkvdjIvIiwiaHR0cHM6Ly9jdWJhcC5hdXRoMC5jb20vdXNlcmluZm8iXSwiaWF0IjoxNzI4Njc1MDczLCJleHAiOjE3Mjg2ODIyNzMsInNjb3BlIjoib3BlbmlkIHByb2ZpbGUgZW1haWwgdXBkYXRlOmN1cnJlbnRfdXNlcl9tZXRhZGF0YSBvZmZsaW5lX2FjY2VzcyIsImF6cCI6ImJCdWdGTVdIVW8xT2huU1pNcFlVWHhpM1kxVUpJN0tsIn0.TRmYqcEIGh5lvt_6mjX6CULsiIthh-P-gWLAl_JDuKjZsbbw-gSzX5ogQ70Iya9vxGdbBemYdKMqW2I8CN7w-4MCt4GMQvLNDXbaf8HUs4ohVqMzkpRvjjqwHtWzrIYO_f8ceKU8KFeu3czGmZ0_1_R30zTrEty0WMEVfhAI1v7qUiE5VY5tef-rbX19lcAMUku4b0xs8lKzlKFzsekSFtZsJ-pOoytMLWoEIeU_hxE8AmdCSm3W9Hg3jAVVIBDMKpY0foI4BSAsCWzfT9qnjBTbKtixM4k4MWyAJISERn9coIJFOG9ae6GP8JiVrIMD9je0gxDSxik8HnRINBsmLA'
```
Comment (kate-holdener) [inline components/classroom/main.js:7]: Generally, it's not a good idea to have tokens in a GitHub repository. @Matthew-Clendenning @MutyaSowmya123 was this an intentional change?

File: components/classroom/groups/api/UpdateProjectData.mjs
Comment (kate-holdener) [inline components/classroom/groups/api/UpdateProjectData.mjs:1]: You deleted this file? intentionally or not?

File: components/classroom/groups/utils/authHelper.mjs
Comment (kate-holdener) [inline components/classroom/groups/utils/authHelper.mjs:1]: This file was deleted, it seems like an accidental change.

File: components/classroom/server.js
Comment (kate-holdener) [inline components/classroom/server.js:1]: This file was deleted, seems like an accidental change.

File: components/classroom/package-lock.json
Comment (kate-holdener) [inline components/classroom/package-lock.json:1]: You probably didn't mean to delete this file.

**Comment by kate-holdener** [comment]
@Matthew-Clendenning @MutyaSowmya123 Several files were deleted with this PR. It seems like an accidental change, which is concerning. Could you clarify?

**Comment by MutyaSowmya123** [comment]
> @Matthew-Clendenning @MutyaSowmya123 Several files were deleted with this PR. It seems like an accidental change, which is concerning. Could you clarify?
We were transitioning from working on MongoDB to the TPEN services API, which required significant restructuring of our codebase. The files deleted were part of that process and were deemed unnecessary for the new implementation. We believe this will ultimately improve the functionality and maintainability of our code.

**Review by MutyaSowmya123 — APPROVED** [review]

**Review by camilledaugherty — APPROVED** [review]

**Pull request merged**